artifactory/cli.go

```diff
@@ -1771,7 +1775,9 @@ func searchCmd(c *cli.Context) {
 	cliutils.ExitOnErr(err)
 	result, err := json.Marshal(searchCmd.SearchResult())
 	cliutils.FailNoOp(err, len(searchCmd.SearchResult()), 0, isFailNoOp(c))
-	log.Output(string(clientutils.IndentJson(result)))
+	if !c.Bool("count") {
```

Comment:
Let's changes this, so that the files / folders count is displayed.

Reply:
Done